composer.json

```diff
@@ -15,7 +15,9 @@
         "symfony/framework-bundle": "^3.4|^4.0",
         "symfony/dependency-injection": "^3.3|^4.0",
         "symfony/http-kernel": "^3.3|^4.0",
-        "doctrine/common": "^2.2"
+        "doctrine/collections": "~1.0",
+        "doctrine/event-manager": "~1.0",
```

Comment:
does this bundle really use the Doctrine events ? AFAIK, it uses `doctrine/annotations` (missing here) and `doctrine/persistence`